File: mintme.py
import sys
from PyQt5.QtCore import QUrl
from PyQt5.QtWidgets import QApplication, QMainWindow
from PyQt5.QtWebEngineWidgets import QWebEngineView, QWebEnginePage
from PyQt5.QtWebEngineCore import QWebEngineSettings

class WebEngineWindow(QMainWindow):
    def __init__(self):
        super().__init__()
        self.browser = QWebEngineView()
        self.browser.setUrl(QUrl("https://www.thesmallflame.com"))  # Thay đổi URL nếu cần
        self.setCentralWidget(self.browser)

        # Chế độ web đầy đủ, bật JavaScript
        self.browser.page().settings().setAttribute(QWebEngineSettings.JavascriptEnabled, True)

        # Tự động tải lại sau một khoảng thời gian nếu cần
        self.setWindowTitle("24h Web Browser")
        self.resize(1024, 768)
        self.show()

if __name__ == "__main__":
    app = QApplication(sys.argv)
    window = WebEngineWindow()

    # Duy trì trang web mở 24 giờ
    sys.exit(app.exec_())  # Chạy ứng dụng Qt cho đến khi đóng

File: mintme.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch({ headless: true }); // hoặc use firefox hoặc webkit
  const page = await browser.newPage();
  await page.goto('https://www.thesmallflame.com');
  console.log("Trang web đang mở và duy trì...");

  // Duy trì trang web mở 24 giờ
  setInterval(async () => {
    console.log("Vẫn giữ trang web mở...");
  }, 3600000); // Mỗi 1 giờ

  // Đợi vô thời hạn
  await page.waitForTimeout(86400000);  // Đợi 24 giờ
  await browser.close();
})();
